pkg/capacityscheduling/capacity_scheduling.go

```diff
@@ -119,42 +126,43 @@ func New(obj runtime.Object, handle framework.Handle) (framework.Plugin, error)
 		pdbLister:         getPDBLister(handle.SharedInformerFactory()),
 	}
 
-	client, err := versioned.NewForConfig(handle.KubeConfig())
-	if err != nil {
+	if client, err := client.New(handle.KubeConfig(), client.Options{Scheme: scheme}); err != nil {
 		return nil, err
+	} else {
+		c.client = client
 	}
 
-	schedSharedInformerFactory := schedinformer.NewSharedInformerFactory(client, 0)
-	c.elasticQuotaLister = schedSharedInformerFactory.Scheduling().V1alpha1().ElasticQuotas().Lister()
-	elasticQuotaInformer := schedSharedInformerFactory.Scheduling().V1alpha1().ElasticQuotas().Informer()
-	elasticQuotaInformer.AddEventHandler(
-		cache.FilteringResourceEventHandler{
-			FilterFunc: func(obj interface{}) bool {
-				switch t := obj.(type) {
-				case *v1alpha1.ElasticQuota:
+	dynamicCache, err := ctrlruntimecache.New(handle.KubeConfig(), ctrlruntimecache.Options{Scheme: scheme})
```

Comment:
Also cc @denkensk, this might need your second look, as I'm changing typed informer to controller-runtime's equivalent (I think ;)) utility. Please shed your light.

Reply:
look good to me.